AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/sortBy.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'Magento_Ui/js/grid/sortBy'
+], function (SortBy) {
```

Comment:
I think it would be better to observe sorting from the preview component and close it when the sorting is appied

Reply:
ok

Reply:
@sivaschenko Done, thank you for review, now it looks much better :+1: